dlt/destinations/filesystem/configuration.py

```diff
@@ -10,6 +10,7 @@
 @configspec
 class FilesystemDestinationClientConfiguration(FilesystemConfiguration, DestinationClientStagingConfiguration): # type: ignore[misc]
     destination_name: Final[str] = "filesystem"  # type: ignore
+    force_iceberg: Optional[bool] = False
```

Comment:
same

Reply:
removed it here as it is no longer needed